google-cloud-storage/src/test/java/com/google/cloud/storage/jqwik/StorageArbitraries.java

```diff
@@ -49,6 +53,20 @@ public static Arbitrary<Timestamp> timestamp() {
                 Timestamp.newBuilder().setSeconds(odt.toEpochSecond()).setNanos(nanos).build());
   }
 
+  public static Arbitrary<Date> date() {
+    return DateTimes.offsetDateTimes()
+        .offsetBetween(ZoneOffset.UTC, ZoneOffset.UTC)
+        .map(
+            odt -> {
+              OffsetDateTime utc = odt.withOffsetSameInstant(ZoneOffset.UTC);
```

Comment:
I don't think you need this transform here, the input has previously been constrained to UTC.